src/components/transactions/LiTransaction.vue

```diff
@@ -20,7 +20,7 @@
       </div>
       <div class="li-tx__content__right">
         <div>
-          Network Fee:&nbsp;<b>{{ fees.amount | toAtoms | shortDecimals }}</b>
+          Network Fee:&nbsp;<b>{{ fees.amount | toAtoms }}</b>
```

Comment:
This is not formatting the number. Was that indented?

Reply:
The new filter defined above only works to group thousands, so it wouldn't apply to network fees, generally speaking.

Short answer, no, but it makes no difference. I can put in anyway at a later commit.